vizro-core/hatch.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
   "coverage[toml]>=6.5",
   "pytest",
   "pytest-mock",
+  "freezegun>=1.5.0",
```

Comment:
The `TickingDateTimeFactory.tick()` (we use it as `freezer.tick()`) [has been supported](https://github.com/spulec/freezegun/commit/e126e111cc64d6da957167bf4ec316266613c687) since version `1.5.0`, hence this limitation.